components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/exceptions/UnsupportedThrottleConditionTypeException.java

```diff
@@ -0,0 +1,14 @@
+package org.wso2.carbon.apimgt.rest.api.admin.exceptions;
```

Comment:
Fixed in https://github.com/wso2/carbon-apimgt/commit/8bf295d69103fd9391adef96f70d05a766cb4879.